src/go/k8s/pkg/resources/certmanager/certificate.go

```diff
@@ -52,6 +55,32 @@ type CertificateResource struct {
 	logger         logr.Logger
 }
 
+// NewCACertificate creates a root or intermediary CA certificate
+func NewCACertificate(
```

Comment:
Because this new function is added the existing one `NewNodeCertificate` is only with isCA bool = false. This function is used once and its `isCA` bool is set to true. Can we get rid of this parameter from both functions?

Reply:
Good catch. I'll update

Reply:
✅